src/test/java/com/google/sps/service/AppConverterTest.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.service;
+
+import com.google.sps.service.AppConverter;
+import com.google.sps.models.App;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public final class AppConverterTest {
+  private static final LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig().setDefaultHighRepJobPolicyUnappliedJobPercentage(0));
+
+    @BeforeClass
+    public static void setup() {
+        helper.setUp();
+    }
+
+    @AfterClass
+    public static void done() {
+        helper.tearDown();
+    }
+
+    @Test
+    public void testAppConverter() throws Exception {
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+        ArrayList<Entity> entities = new ArrayList<Entity>();
+        Entity app1 = new Entity("App");
+        app1.setProperty("category", "test");
+        app1.setProperty("description", "this is a test app");
+        app1.setProperty("url", "www.testapp.com");
+        app1.setProperty("id", "testId");
+        app1.setProperty("title", "test app 1");
+        app1.setProperty("price", 0.99);
+        app1.setProperty("rating", 3.6); 
+        app1.setProperty("rated", "Everyone");
+        datastore.put(app1);  
+
+        Entity app2 = new Entity("App");
+        app2.setProperty("category", "test");
+        app2.setProperty("description", "this is another test app");
+        app2.setProperty("url", "www.testapp2.com");
+        app2.setProperty("id", "test2Id");
+        app2.setProperty("title", "test app 2");
+        app2.setProperty("price", 0.99);
+        app2.setProperty("rating", 3.6); 
+        app2.setProperty("rated", "Everyone");
+        datastore.put(app2); 
+
+        entities.add(app1);
+        entities.add(app2);
+
+        List<App> convertedApps = new AppConverter().convertToApp(entities);
```

Comment:
Can you also verify there's 2 items in the list.

It might be better to construct your expected Apps and compare against that.